README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
+- If we close the application when we open it again we can continue from where we were
+- Flutter benefits from several objects that help us to facilitate the construction process
+
+## Core Principles
+
+Flutter is designed for the creation of 2D mobile applications.</br> 
+It contains several widgets and tools that facilitate the realization of an app.</br>
+
+## Widgets
+
+Widget is a component that can be used to simplify the process of building applications.</br>
+It usually contains windows, buttons, icons, menus, scroll bars and other features.</br>
+
+A widget can define:
+- Physical elements of an application (buttons, menus or bars)
+- Visual elements such as colors  
+- Elements of the application aspect such as edges or the padding of the screen
+
+Widgets form a hierarchy whenever we use one of them we will have another that will complement it.</br>
```

Comment:
```suggestion
Widgets are assembled in declarative hierarchy which allows us to easily organise the layout of our App as a series of nested widgets.</br>
```
This sentence does not make much sense.  I know what you _mean_ to say here, but the sentence does not convey your intended meaning. Yes, there is a hierarchy and widgets are nested but "complement it" is out of place.

Reply:
As I wrote before, I still have to practice the English very well. 😄